src/components/ZetkinCalendar/index.tsx

```diff
@@ -51,10 +72,42 @@ const ZetkinCalendar = ({ baseHref, events, campaigns , tasks }: ZetkinCalendarP
         }
     };
 
+    const isTodayBeforeView = () =>{
+        if (focusDate.getFullYear() > dayjs().get('year')) return true;
+        if (focusDate.getFullYear() < dayjs().get('year')) return false;
+        //only needed if same year
+        if (range === CALENDAR_RANGES.MONTH) {
+            if (focusDate.getMonth() > dayjs().get('month')) return true;
+        }
+        else if (range === CALENDAR_RANGES.WEEK) {
+            if (focusDate.getWeekNumber() > dayjs().week()) return true;
+        }
+        return false;
+    };
+
+    const isTodayAfterView = () =>{
```

Comment:
Using @richardolsson 's suggestion above, you'd cut down on a lot of code by doing 
`isTodayBeforeView = today < firstCalendarDate` (i think this what the variable is called, it's the first date in the view)
`isTodayAfterView = today > lastCalendarDate`

Reply:
The problem is that those variables are only available inside the distinct views, i.e. the month and week calendars respectively (and in fact the `lastCalendarDate` does not exist). So they would need to be calculated here too, e.g. using some utility function.

Reply:
Yes, sorry, use utility functions and then do: 

`isTodayBeforeView = today < getfirstDateInView()`

Reply:
Me and @omidhq solved this together in 9dc6847113d5e2215abe6aa2e2d89f3d295a2558.